detox/detox.d.ts

```diff
@@ -1112,6 +1112,31 @@ declare global {
              * web.element(by.web.tag('mark'))
              */
             tag(tagName: string): WebMatcher;
+
+            /**
+             * (iOS Only) Find an element on the DOM tree by its value
+             * @param value
+             * @example
+             * web.element(by.web.value('hello'))
+             */
+            value(value: string): WebMatcher;
```

Comment:
Sounds like `<input value="...">` right? Why is it iOS-only, then? 🤔

Reply:
We should have it for Android as well. As soon as we rewrite the infra to use same JS code builder for Android + iOS, we'll have this API there as well. Right now, we just wrap Espresso on Android, and it doesn't have it

Reply:
actually, we don't have to wait and we can simply wrap runscript from the javascript side for Android and support it there as well, however there are many other things that can be done like this, and I prefer to keep it and do it properly for all